resolwe/flow/executors/prepare.py

```diff
@@ -0,0 +1,49 @@
+"""Base class for the manager-internal preparation bits of the executor."""
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import logging
+import os
+
+from django.apps import apps
+from django.conf import settings
+from django.forms.models import model_to_dict
+
+from resolwe.flow.models import Data
+
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+CWD = os.getcwd()
+
+
+class BaseFlowExecutorPreparer(object):
+    """Represents the preparation functionality of the workflow executor."""
+
+    def __init__(self, *args, **kwargs):
+        """Initialize attributes."""
+
+    def make_settings(self, data_id):
+        """Add any additional required settings into what the manager will serialize."""
+        return {
+            'USE_TZ': settings.USE_TZ,
+            'FLOW_EXECUTOR_TOOLS_PATHS': self.get_tools(),
+            'Data_get': model_to_dict(Data.objects.get(
+                pk=data_id
+            )),
+            'Process_get': model_to_dict(Data.objects.get(
+                pk=data_id
+            ).process),
+        }
+
+    def get_tools(self):
+        """Get tools paths."""
+        tools_paths = []
+        for app_config in apps.get_app_configs():
+            proc_path = os.path.join(app_config.path, 'tools')
```

Comment:
Can `'tools'` be moved into a constant and used instead?

Reply:
To be tackled later, as discussed.